### PR TITLE
add indianhealthyrecipes

### PR DESCRIPTION
**Pull request by bfcarpio**
Just a parser this time :)

Part of #206

### CODE REVIEW

File: tests/test_indianhealthyrecipes.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        self.assertEqual("16 serving(s)", self.harvester_class.yields())
+
+    def test_image(self):
+        self.assertEqual(
+            "https://www.indianhealthyrecipes.com/wp-content/uploads/2015/12/banana-cake-2.jpg",
+            self.harvester_class.image(),
+        )
+
+    def test_ingredients(self):
+        self.assertCountEqual(
```
[the remark this answers is not in the document]
Comment (bfcarpio) [reply tests/test_indianhealthyrecipes.py]: Fixed!

**Review by jayaddison — APPROVED** [review]
Thanks!

**Review by hhursev — APPROVED** [review]
README.rst addition missing

**Comment by bfcarpio** [comment]
Whoops, thanks for the catch. I thought the generator would handle it. Expect another PR with the other 2 I missed.

**Pull request merged**